components/consumers/pdf/README.md

```diff
@@ -0,0 +1,39 @@
+# PDF consumer
+
+This consumer prints the pipeline results into a Go template, prints them into a
+PDF and uploads them into an S3 bucket.
```

Comment:
⛏️ we should say that this last step is optional

Reply:
The last step isn't optional, the "skipS3" flag is for development purposes only.